website/docs/en/config/stats.mdx

```diff
@@ -36,167 +36,415 @@ Generate packaging information that can be used to analyze module dependencies a
 | `'detailed'`        | Output everything except `chunkModules` and `chunkRootModules` |
 | `'summary'`         | Output only the summary information                            |
 
-## Stats Options
-
 You can specify exactly which packing information to output, all the following fields are optional.
 
+## Preset Options
+
+### stats.all
+
+<PropertyType
+  type="boolean"
+  defaultValueList={[{ defaultValue: 'undefined' }]}
+/>
+
+A fallback value for stats options when an option is not defined. It has precedence over local Rspack defaults.
```

Comment:
we should tell users that `stats.all` is quite slow and they should avoid using it.